privaterelay/apps.py

```diff
@@ -17,6 +18,43 @@
     from allauth.socialaccount.models import SocialApp
 
 
+def configure_google_profiler() -> None:
+    if (
+        settings.GOOGLE_APPLICATION_CREDENTIALS == ""
+        or settings.GOOGLE_CLOUD_PROFILER_CREDENTIALS_B64 == ""
+    ):
+        return
+
+    # Set up Google Cloud Profiler
+    service, version = get_profiler_startup_data()
+    if service is None:
+        return
+
+    gcp_key_json_path = Path(settings.GOOGLE_APPLICATION_CREDENTIALS)
+    if not gcp_key_json_path.exists():
+        write_gcp_key_json_file(gcp_key_json_path)
+    try:
+        with gcp_key_json_path.open() as gcp_key_file:
+            try:
+                # Make sure the expect gcp_key.json file is valid json
```

Comment:
quibble (non-blocking): 
```suggestion
                # Make sure the expected gcp_key.json file is valid json
```